1-getting-started/Translations/README.fr.md

```diff
@@ -0,0 +1,15 @@
+# Guide de démarage IoT
+Dans cette section du tutoriel, vous serez initié à l'Internet des objets et apprendrez les concepts de base, notamment la construction de votre premier projet IoT « Hello, World!» connecté à l'infrastrucutre infonuagique. Ce projet est une veilleuse qui s'allume quand la lumière mesurée par un capteur diminue.
```

Comment:
Small typo to fix on line 2 :
" ... connecté à l'infrastrucutre infonuagique." 
It should be *"infrastructure"* we could also simply say "... connecté au Cloud".